src/login/logind-button.c

```diff
@@ -158,6 +159,14 @@ static int button_dispatch(sd_event_source *s, int fd, uint32_t revents, void *u
                         manager_handle_action(b->manager, INHIBIT_HANDLE_POWER_KEY, b->manager->handle_power_key, b->manager->power_key_ignore_inhibited, true);
                         break;
 
+                case KEY_RESTART:
+                        log_struct(LOG_INFO,
+                                   LOG_MESSAGE("Restart key pressed."),
+                                   "MESSAGE_ID=" SD_MESSAGE_RESTART_KEY_STR);
```

Comment:
Please add comment here about confused kernel naming pointing out the fact that KEY_RESTART probably was introduced for media playback purposes, but is apparently predominantly used as reboot key.